lib/pages/child_menu/customizable_column.dart

```diff
@@ -89,15 +97,18 @@ class _CustomizableColumnState extends State<CustomizableColumn> {
   Future<void> submit(BuildContext context) async {
```

Comment:
Please rename this method to something more meaningful

Reply:
Please also try to divide this method in multiple smaller ones as it is very long